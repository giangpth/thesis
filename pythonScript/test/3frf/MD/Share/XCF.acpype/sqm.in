Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   7    N4       26.6790        7.1740       40.9960 
   6    C3       27.2100        8.4150       41.0190 
   7    N5       26.7380        9.4090       40.1450 
   7    N2       28.2380        8.7360       41.9090 
   6    C1       28.8210       10.0210       41.9210 
   6    C8       28.3810       10.9860       41.0610 
   6    C6       27.3100       10.6860       40.1670 
   7    N7       26.7720       11.5660       39.2980 
   6    C9       28.9970       12.3640       41.0030 
   6   C10       30.0310       12.5930       42.0740 
   6   C11       31.3360       12.0910       41.9410 
   6   C27       31.7420       11.3500       40.7550 
   6   C28       32.9250       10.7360       40.6940 
   6   C14       33.8870       10.7720       41.8460 
   6   C24       35.3250       10.9980       41.3470 
   6   C25       36.3970       11.3270       42.3450 
   6   C26       35.9080       12.3820       41.3930 
   8   O13       33.5200       11.7100       42.8690 
   6   C12       32.2710       12.2990       42.9680 
   6   C15       31.9280       13.0740       44.0890 
   8   O16       32.7730       13.3680       45.1500 
   6   C17       34.0950       13.8570       44.9300 
   6   C18       30.6260       13.5590       44.2010 
   6   C21       29.6800       13.3190       43.2040 
   8   O19       30.3110       14.3060       45.3150 
   6   C20       29.0150       14.8910       45.3260 
   1    H1       29.6190       10.2420       42.6140 
   1   H91       29.4710       12.4910       40.0300 
   1   H92       28.2080       13.1080       41.1110 
   1   H27       31.0730       11.2960       39.9090 
   1   H28       33.1970       10.2010       39.7960 
   1   H14       33.8650        9.7830       42.3040 
   1   H24       35.6360       10.3810       40.5040 
   1  H251       36.1520       11.3940       43.4050 
   1  H252       37.3820       10.9190       42.1200 
   1  H261       36.5530       12.7290       40.5860 
   1  H262       35.2640       13.1430       41.8330 
   1   H21       28.6740       13.6980       43.3110 
   1   H41       25.9360        6.9540       40.3480 
   1   H42       27.0240        6.4650       41.6270 
   1   H71       26.0140       11.2810       38.6950 
   1   H72       27.1270       12.5100       39.2500 
   1  H171       34.3230       14.6320       45.6620 
   1  H172       34.8070       13.0380       45.0340 
   1  H173       34.1660       14.2740       43.9250 
   1  H201       28.8930       15.4880       46.2300 
   1  H202       28.8970       15.5290       44.4500 
   1  H203       28.2610       14.1040       45.3060 

