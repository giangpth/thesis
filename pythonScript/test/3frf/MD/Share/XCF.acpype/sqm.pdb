ATOM      1  N4  XCF   300      26.050   7.551  41.193  1.00  0.00           N
ATOM      2  C3  XCF   300      26.788   8.737  41.151  1.00  0.00           C
ATOM      3  N5  XCF   300      26.622   9.548  40.044  1.00  0.00           N
ATOM      4  N2  XCF   300      27.666   8.989  42.203  1.00  0.00           N
ATOM      5  C1  XCF   300      28.430  10.080  42.076  1.00  0.00           C
ATOM      6  C8  XCF   300      28.391  10.959  40.980  1.00  0.00           C
ATOM      7  C6  XCF   300      27.443  10.640  39.949  1.00  0.00           C
ATOM      8  N7  XCF   300      27.378  11.413  38.790  1.00  0.00           N
ATOM      9  C9  XCF   300      29.262  12.154  40.877  1.00  0.00           C
ATOM     10  C10 XCF   300      30.237  12.315  41.997  1.00  0.00           C
ATOM     11  C11 XCF   300      31.481  11.666  41.980  1.00  0.00           C
ATOM     12  C27 XCF   300      31.880  10.734  40.943  1.00  0.00           C
ATOM     13  C28 XCF   300      33.127  10.245  40.922  1.00  0.00           C
ATOM     14  C14 XCF   300      34.151  10.634  41.930  1.00  0.00           C
ATOM     15  C24 XCF   300      35.248  11.450  41.288  1.00  0.00           C
ATOM     16  C25 XCF   300      36.299  12.105  42.144  1.00  0.00           C
ATOM     17  C26 XCF   300      35.302  12.948  41.407  1.00  0.00           C
ATOM     18  O13 XCF   300      33.637  11.304  43.100  1.00  0.00           O
ATOM     19  C12 XCF   300      32.401  11.903  43.026  1.00  0.00           C
ATOM     20  C15 XCF   300      32.067  12.746  44.105  1.00  0.00           C
ATOM     21  O16 XCF   300      32.879  12.858  45.218  1.00  0.00           O
ATOM     22  C17 XCF   300      34.015  13.702  45.017  1.00  0.00           C
ATOM     23  C18 XCF   300      30.811  13.391  44.097  1.00  0.00           C
ATOM     24  C21 XCF   300      29.908  13.172  43.051  1.00  0.00           C
ATOM     25  O19 XCF   300      30.577  14.237  45.160  1.00  0.00           O
ATOM     26  C20 XCF   300      29.249  14.736  45.283  1.00  0.00           C
ATOM     27  H1  XCF   300      29.130  10.267  42.916  1.00  0.00           H
ATOM     28  H91 XCF   300      29.832  12.120  39.905  1.00  0.00           H
ATOM     29  H92 XCF   300      28.610  13.074  40.836  1.00  0.00           H
ATOM     30  H27 XCF   300      31.122  10.431  40.204  1.00  0.00           H
ATOM     31  H28 XCF   300      33.469   9.535  40.156  1.00  0.00           H
ATOM     32  H14 XCF   300      34.616   9.703  42.381  1.00  0.00           H
ATOM     33  H24 XCF   300      35.565  10.994  40.328  1.00  0.00           H
ATOM     34 H251 XCF   300      36.211  12.015  43.242  1.00  0.00           H
ATOM     35 H252 XCF   300      37.349  12.088  41.805  1.00  0.00           H
ATOM     36 H261 XCF   300      35.630  13.538  40.533  1.00  0.00           H
ATOM     37 H262 XCF   300      34.510  13.463  41.983  1.00  0.00           H
ATOM     38  H21 XCF   300      28.931  13.675  43.050  1.00  0.00           H
ATOM     39  H41 XCF   300      25.267   7.484  40.587  1.00  0.00           H
ATOM     40  H42 XCF   300      25.971   7.104  42.077  1.00  0.00           H
ATOM     41  H71 XCF   300      26.612  11.250  38.177  1.00  0.00           H
ATOM     42  H72 XCF   300      27.641  12.364  38.901  1.00  0.00           H
ATOM     43 H171 XCF   300      34.541  13.671  46.004  1.00  0.00           H
ATOM     44 H172 XCF   300      34.663  13.290  44.205  1.00  0.00           H
ATOM     45 H173 XCF   300      33.689  14.741  44.773  1.00  0.00           H
ATOM     46 H201 XCF   300      29.275  15.295  46.254  1.00  0.00           H
ATOM     47 H202 XCF   300      29.012  15.427  44.438  1.00  0.00           H
ATOM     48 H203 XCF   300      28.510  13.901  45.328  1.00  0.00           H
