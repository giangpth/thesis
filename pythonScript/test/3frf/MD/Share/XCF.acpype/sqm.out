            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 68

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      126
| QMMM:    diag iterations used for timing =       10
| QMMM:
| QMMM:              Internal diag routine = 0.204861 seconds
| QMMM:                 Dspev diag routine = 0.394671 seconds
| QMMM:                Dspevd diag routine = 0.174689 seconds
| QMMM:                Dspevx diag routine = 0.356700 seconds
| QMMM:                 Dsyev diag routine = 0.195850 seconds
| QMMM:                Dsyevd diag routine = 0.041873 seconds
| QMMM:                Dsyevr diag routine = 0.066050 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.036032 seconds
| QMMM:
| QMMM: Using dsyevd routine (diag_routine=6).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N       26.6790    7.1740   40.9960
  QMMM:     2        2      C       27.2100    8.4150   41.0190
  QMMM:     3        3      N       26.7380    9.4090   40.1450
  QMMM:     4        4      N       28.2380    8.7360   41.9090
  QMMM:     5        5      C       28.8210   10.0210   41.9210
  QMMM:     6        6      C       28.3810   10.9860   41.0610
  QMMM:     7        7      C       27.3100   10.6860   40.1670
  QMMM:     8        8      N       26.7720   11.5660   39.2980
  QMMM:     9        9      C       28.9970   12.3640   41.0030
  QMMM:    10       10      C       30.0310   12.5930   42.0740
  QMMM:    11       11      C       31.3360   12.0910   41.9410
  QMMM:    12       12      C       31.7420   11.3500   40.7550
  QMMM:    13       13      C       32.9250   10.7360   40.6940
  QMMM:    14       14      C       33.8870   10.7720   41.8460
  QMMM:    15       15      C       35.3250   10.9980   41.3470
  QMMM:    16       16      C       36.3970   11.3270   42.3450
  QMMM:    17       17      C       35.9080   12.3820   41.3930
  QMMM:    18       18      O       33.5200   11.7100   42.8690
  QMMM:    19       19      C       32.2710   12.2990   42.9680
  QMMM:    20       20      C       31.9280   13.0740   44.0890
  QMMM:    21       21      O       32.7730   13.3680   45.1500
  QMMM:    22       22      C       34.0950   13.8570   44.9300
  QMMM:    23       23      C       30.6260   13.5590   44.2010
  QMMM:    24       24      C       29.6800   13.3190   43.2040
  QMMM:    25       25      O       30.3110   14.3060   45.3150
  QMMM:    26       26      C       29.0150   14.8910   45.3260
  QMMM:    27       27      H       29.6190   10.2420   42.6140
  QMMM:    28       28      H       29.4710   12.4910   40.0300
  QMMM:    29       29      H       28.2080   13.1080   41.1110
  QMMM:    30       30      H       31.0730   11.2960   39.9090
  QMMM:    31       31      H       33.1970   10.2010   39.7960
  QMMM:    32       32      H       33.8650    9.7830   42.3040
  QMMM:    33       33      H       35.6360   10.3810   40.5040
  QMMM:    34       34      H       36.1520   11.3940   43.4050
  QMMM:    35       35      H       37.3820   10.9190   42.1200
  QMMM:    36       36      H       36.5530   12.7290   40.5860
  QMMM:    37       37      H       35.2640   13.1430   41.8330
  QMMM:    38       38      H       28.6740   13.6980   43.3110
  QMMM:    39       39      H       25.9360    6.9540   40.3480
  QMMM:    40       40      H       27.0240    6.4650   41.6270
  QMMM:    41       41      H       26.0140   11.2810   38.6950
  QMMM:    42       42      H       27.1270   12.5100   39.2500
  QMMM:    43       43      H       34.3230   14.6320   45.6620
  QMMM:    44       44      H       34.8070   13.0380   45.0340
  QMMM:    45       45      H       34.1660   14.2740   43.9250
  QMMM:    46       46      H       28.8930   15.4880   46.2300
  QMMM:    47       47      H       28.8970   15.5290   44.4500
  QMMM:    48       48      H       28.2610   14.1040   45.3060

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10        8.3281 kcal/mol        0.6465 kcal/(mol*A)
xmin    20        7.7584 kcal/mol        0.5978 kcal/(mol*A)
xmin    30        7.3816 kcal/mol        0.1821 kcal/(mol*A)
xmin    40        7.3042 kcal/mol        0.1034 kcal/(mol*A)
xmin    50        7.2662 kcal/mol        0.0647 kcal/(mol*A)
xmin    60        7.1181 kcal/mol        0.1954 kcal/(mol*A)
xmin    70        7.0157 kcal/mol        0.0706 kcal/(mol*A)
xmin    80        6.9564 kcal/mol        0.1171 kcal/(mol*A)
xmin    90        6.9508 kcal/mol        0.0059 kcal/(mol*A)
xmin   100        6.9504 kcal/mol        0.0089 kcal/(mol*A)
xmin   110        6.9503 kcal/mol        0.0010 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -41.6770  -40.4760  -40.1184  -38.5305  -38.3254  -37.2383  -36.5753  -34.9754
 -34.5436  -33.2527  -32.1087  -31.0433  -30.0481  -28.0066  -27.5309  -27.0226
 -26.1614  -24.3008  -23.9744  -23.2289  -23.1419  -22.1023  -21.6655  -19.9614
 -19.2676  -18.8708  -18.6911  -18.3011  -17.5739  -17.1130  -17.0777  -16.9253
 -16.6894  -16.1984  -16.0976  -15.8082  -15.7395  -15.5572  -15.4953  -15.4196
 -15.2719  -14.9093  -14.7688  -14.1268  -14.0344  -13.9488  -13.6167  -13.4874
 -12.9989  -12.8746  -12.7207  -12.6392  -12.4548  -12.3670  -12.2438  -12.1921
 -11.9341  -11.7586  -11.5087  -11.4871  -11.0135  -10.6353  -10.4895  -10.2000
 -10.0639   -8.9940   -8.7428   -8.5250   -0.1937    0.2176    0.2691    0.4344
   1.3724    1.9732    2.0319    2.5689    2.6467    2.7282    2.7625    2.8806
   2.9749    3.3592    3.4817    3.5796    3.6286    3.6949    3.7807    3.8328
   3.8640    3.9182    3.9475    4.0428    4.1030    4.1673    4.2006    4.2192
   4.2474    4.3193    4.3626    4.3975    4.4679    4.4961    4.5852    4.6663
   4.7081    4.7384    4.8056    4.8533    4.9159    5.0631    5.0770    5.1111
   5.1258    5.1994    5.3296    5.3700    5.5193    5.7376    5.7735    5.9730
   6.0581    6.1229    6.3600    6.3809    6.7473    7.2720

 Heat of formation   =          6.95028997 kcal/mol  (        0.30138719 eV)

 Total SCF energy    =    -104274.90719776 kcal/mol  (    -4521.69928441 eV)
 Electronic energy   =    -805223.87266859 kcal/mol  (   -34917.12730014 eV)
 Core-core repulsion =     700948.96547083 kcal/mol  (    30395.42801573 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      N                 -0.328
     2      C                  0.178
     3      N                 -0.293
     4      N                 -0.247
     5      C                  0.031
     6      C                 -0.289
     7      C                  0.179
     8      N                 -0.335
     9      C                 -0.059
    10      C                  0.002
    11      C                 -0.136
    12      C                 -0.088
    13      C                 -0.203
    14      C                  0.095
    15      C                 -0.193
    16      C                 -0.204
    17      C                 -0.199
    18      O                 -0.196
    19      C                  0.113
    20      C                 -0.015
    21      O                 -0.191
    22      C                 -0.078
    23      C                  0.106
    24      C                 -0.232
    25      O                 -0.189
    26      C                 -0.079
    27      H                  0.175
    28      H                  0.100
    29      H                  0.079
    30      H                  0.147
    31      H                  0.141
    32      H                  0.120
    33      H                  0.125
    34      H                  0.121
    35      H                  0.108
    36      H                  0.109
    37      H                  0.117
    38      H                  0.142
    39      H                  0.222
    40      H                  0.225
    41      H                  0.215
    42      H                  0.206
    43      H                  0.099
    44      H                  0.075
    45      H                  0.067
    46      H                  0.109
    47      H                  0.069
    48      H                  0.078
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -2.228    2.583   -2.662    4.327
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      N       26.0496    7.5514   41.1934
  QMMM:     2        2      C       26.7875    8.7368   41.1514
  QMMM:     3        3      N       26.6218    9.5483   40.0438
  QMMM:     4        4      N       27.6665    8.9894   42.2034
  QMMM:     5        5      C       28.4305   10.0803   42.0764
  QMMM:     6        6      C       28.3911   10.9592   40.9800
  QMMM:     7        7      C       27.4433   10.6400   39.9490
  QMMM:     8        8      N       27.3785   11.4127   38.7904
  QMMM:     9        9      C       29.2616   12.1540   40.8766
  QMMM:    10       10      C       30.2370   12.3145   41.9973
  QMMM:    11       11      C       31.4814   11.6659   41.9803
  QMMM:    12       12      C       31.8796   10.7337   40.9433
  QMMM:    13       13      C       33.1269   10.2452   40.9215
  QMMM:    14       14      C       34.1514   10.6345   41.9301
  QMMM:    15       15      C       35.2482   11.4497   41.2882
  QMMM:    16       16      C       36.2989   12.1052   42.1439
  QMMM:    17       17      C       35.3020   12.9485   41.4071
  QMMM:    18       18      O       33.6366   11.3041   43.0996
  QMMM:    19       19      C       32.4007   11.9032   43.0263
  QMMM:    20       20      C       32.0672   12.7465   44.1051
  QMMM:    21       21      O       32.8794   12.8583   45.2175
  QMMM:    22       22      C       34.0147   13.7017   45.0174
  QMMM:    23       23      C       30.8114   13.3906   44.0972
  QMMM:    24       24      C       29.9078   13.1717   43.0507
  QMMM:    25       25      O       30.5771   14.2366   45.1595
  QMMM:    26       26      C       29.2494   14.7362   45.2835
  QMMM:    27       27      H       29.1298   10.2672   42.9163
  QMMM:    28       28      H       29.8322   12.1201   39.9051
  QMMM:    29       29      H       28.6103   13.0744   40.8355
  QMMM:    30       30      H       31.1221   10.4315   40.2037
  QMMM:    31       31      H       33.4688    9.5348   40.1557
  QMMM:    32       32      H       34.6158    9.7033   42.3813
  QMMM:    33       33      H       35.5654   10.9936   40.3284
  QMMM:    34       34      H       36.2111   12.0149   43.2424
  QMMM:    35       35      H       37.3494   12.0875   41.8053
  QMMM:    36       36      H       35.6302   13.5380   40.5333
  QMMM:    37       37      H       34.5104   13.4625   41.9830
  QMMM:    38       38      H       28.9306   13.6752   43.0499
  QMMM:    39       39      H       25.2668    7.4838   40.5869
  QMMM:    40       40      H       25.9713    7.1044   42.0766
  QMMM:    41       41      H       26.6123   11.2504   38.1768
  QMMM:    42       42      H       27.6411   12.3643   38.9013
  QMMM:    43       43      H       34.5410   13.6708   46.0035
  QMMM:    44       44      H       34.6631   13.2904   44.2047
  QMMM:    45       45      H       33.6890   14.7410   44.7729
  QMMM:    46       46      H       29.2753   15.2948   46.2537
  QMMM:    47       47      H       29.0115   15.4272   44.4384
  QMMM:    48       48      H       28.5101   13.9006   45.3282

           --------- Calculation Completed ----------

